src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -144,14 +147,16 @@
     </member>
     <member name="List.TrueForAll">
       <summary>Returns true if all items in the list evaluate to true with the given predicate.</summary>
-      <param name="list">list of values</param>
-      <param name="predicate"></param>
+      <param name="list">List from which to evaluate items by query function</param>
+      <param name="queryFunction">Function to evaluate list</param>
+      <returns name="bool">Returns true for all of the items in the list compared to the query function</returns>
       <search>true,all</search>
     </member>
     <member name="List.TrueForAny">
       <summary>Returns true if one or more items in the list evaluate to true with the given predicate.</summary>
-      <param name="list">list of values</param>
-      <param name="predicate"></param>
+      <param name="list">List from which to evaluate items by query function</param>
+      <param name="queryFunction">Function to evaluate list</param>
+      <returns name="bool">Returns true if any of the items in the list compared to the query function</returns>
```

Comment:
NIT: The second part of my previous comment applies here too.

Reply:
Addressed in 2a16aa5